releasenotes/notes/drop_deprecated-7ea90b212509b082.yaml

```diff
@@ -0,0 +1,5 @@
+---
+deprecations:
+  - "`BaseRetrying.call` was long time deprecated and produced `DeprecationWarning`"
+  - "`BaseRetrying.fn` was noted as deprecated"
+  - "`BaseRetrying.begin()` do not require arguments anymore as it not setting `BaseRetrying.fn`"
```

Comment:
You need to put that in the `upgrade` section as "those has been removed".
It's not clear for users reading this that they got removed.